script/server_bootstrap.sh

```diff
@@ -36,7 +37,7 @@ cd $HOME
 git clone https://github.com/yihangho/SimpleCMS.git --bare
 cd SimpleCMS.git
 mkdir -p $DEPLOYMENT_DIR
-GIT_WORK_TREE=$DEPLOYMENT_DIR git checkout -f master
+GIT_WORK_TREE=$DEPLOYMENT_DIR git checkout -f mcc2015
```

Comment:
I am not sure if pushing to master is correct, since master is always the dev branch. Perhaps a good time to checkout a tag?